app/models/null/leaderboard_row.rb

```diff
@@ -0,0 +1,36 @@
+module Null
+  class LeaderboardRow
+    attr_reader :player
+
+    def initialize(player)
+      @player = player
+    end
+
+    def position
+    end
+
+    def played
+      0
+    end
+
+    def corp_wins
+      0
+    end
+
+    def runner_wins
+      0
+    end
+
+    def participation_points
+      0
+    end
+
+    def achievement_points
+      0
+    end
+
+    def points
+      0
+    end
+  end
+end
```

Comment:
Not sure what the ideal structure is really, I just created identically named classes in a `Null` namespace, which also allowed me to tidy them away into a dedicated folder. Seemed clean, but I'm mostly making it up!